kubernetes/annotations_and_labels.md

```diff
@@ -66,57 +66,62 @@ This page defines common annotations and labels we set in Kubernetes objects.
 
 ### Annotations and Labels Set In Cluster API Custom Resource Objects
 
-#### App Catalog
+#### App/AppCatalog
 
-##### Labels
+- Annotation
+    - `giantswarm.io/monitoring` - `true|false', indicating monitoring feature on/off on the service
+    - `giantswarm.io/monitoring-path` - value should indicate monitoring path
+    - `giantswarm.io/monitoring-port` - value should indicate monitoring port on the specific path
 
-- `application.giantswarm.io/catalog-type`
-  Describes the type of catalog that this AppCatalog CR represents.
-  This label helps our UI determine how to show this app catalog.
+- Labels
 
-  - `internal` - Will not show up in our UIs at all.
-  - `managed` - Will gain a 'managed' banner, helping it stand out from other catalogs.
-  - `incubator` - Will show some expectation management message before installing an app from this catalog, that it is still a work in progress, but expected to at least install and somewhat work.
-  - `community` - Will show a more strongly worded expectation management message, indicating that  apps from this catalog will most likely not work without some adjustments.
+    - `application.giantswarm.io/catalog-type`
+      Describes the type of catalog that this AppCatalog CR represents.
+      This label helps our UI determine how to show this app catalog.
+
+      - `internal` - Will not show up in our UIs at all.
+      - `managed` - Will gain a 'managed' banner, helping it stand out from other catalogs.
+      - `incubator` - Will show some expectation management message before installing an app from this catalog, that it is still a work in progress, but expected to at least install and somewhat work.
+      - `community` - Will show a more strongly worded expectation management message, indicating that  apps from this catalog will most likely not work without some adjustments.
 
 #### Cluster
 
-##### Annotations
+- Annotations
```

Comment:
Let's please not change formatting in the PR introducing new labels.

Reply:
Reverted, and adding only specs now.